packages/patternfly-4/react-core/src/components/Progress/Progress.tsx

```diff
@@ -116,8 +112,3 @@ class Progress extends Component {
     );
```

Comment:
Instead of making a type like Joachim suggests, `ariaProps={ariaProps}` could really just be `{...ariaProps}` on line 109. Then, in child components you could use `{...props}` instead of `{...ariaProps}`. This isn't a change for consumers since only the `Progress` is exported for them.